ecc/bw6-761/fr/polynomial/polynomial.go

```diff
@@ -121,3 +147,75 @@ func (p *Polynomial) Equal(p1 Polynomial) bool {
 
 	return true
 }
+
+func signedBigInt(v *fr.Element) big.Int {
```

Comment:
It does!
https://github.com/ConsenSys/gnark-crypto/blob/master/ecc/bn254/fp/element.go#L824

Also, in this case using a pool makes sense because `big.Int.SetBytes` does sometimes reuse previously allocated memory correct?

Reply:
yes correct . We should by the way probably mutualize the big int pool declared in fr/ fp/ on each curves. Will create a ticket 👍